scripts/check-manifest.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
 ./scripts/create-manifest.py
+if [ ! $? -eq 0 ]; then
```

Comment:
There is not-equal operator as well: `-ne`.
`[ ]` is shell test, `[[ ]]` is bash test (preferred).

Reply:
Thank you, solved.